packages/truffle-workflow-compile/index.js

```diff
@@ -70,7 +70,7 @@ var Contracts = {
     const compilers = (config.compiler)
       ? [config.compiler]
       : Object.keys(config.compilers);
-
+      
```

Comment:
Can you remove the whitespace here?

Reply:
Should really integrate prettier into the workflow. Someone with more bandwidth should get on that.